```
======================================================================
FAIL: test_push_inbound_sms (corehq.apps.sms.tests.test_backends.AllBackendTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/mnt/corehq/apps/sms/tests/test_backends.py", line 370, in test_push_inbound_sms
    content_type='application/xml', payload=xml)
  File "/mnt/corehq/apps/sms/tests/test_backends.py", line 192, in _simulate_inbound_request_with_payload
    self.assertEqual(response.status_code, 200)
AssertionError: 400 != 200
----------------------------------------------------------------------
```
failure looks real

--- a/corehq/apps/sms/tests/test_backends.py
+++ b/corehq/apps/sms/tests/test_backends.py
@@ -358,8 +358,7 @@ class AllBackendTest(BaseSMSTest):
         user.delete()
 
     def test_push_inbound_sms(self):
-        xml = """
-        <?xml version="1.0" encoding="UTF-8"?>
+        xml = """<?xml version="1.0" encoding="UTF-8"?>
         <bspostevent>
             <field name="MobileNumber" type="string">99912345</field>
             <field name="Text" type="string">push test</field>

